scripts/codegen.sh

```
@@ -0,0 +1,21 @@
+#!/bin/bash
+set -euo pipefail
+
+SCRIPT_DIR=$(CDPATH= cd -- "$(dirname -- "${BASH_SOURCE[0]}")" && pwd)
```

Shellcheck (0.11.0)

[warning] Remove space after = if trying to assign a value (for empty string, use var='' ... ).

(SC1007)